pkg/cloudprovider/cloud.go

```diff
@@ -37,6 +37,8 @@ type Interface interface {
 	Clusters() (Clusters, bool)
 	// Routes returns a routes interface along with whether the interface is supported.
 	Routes() (Routes, bool)
+	// Volumes returns a volumes interface along with whether the interface is supported.
+	Volumes() (Volumes, bool)
```

Comment:
If I run my own NFS or iSCSI or gluster or ceph then yeah, my GCE cluster supports multiple kinds of volumes.